cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -296,42 +189,24 @@ private ResourceOperatorSupplier supplierWithMocks() {
                 ResourceUtils.metricsProvider(), new PlatformFeaturesAvailability(false, kubernetesVersion), 2_000);
     }
 
-    private void assertResourceRequirements(VertxTestContext context, String statefulSetName) {
-        context.verify(() -> {
-            StatefulSet statefulSet = client.apps().statefulSets().inNamespace(NAMESPACE).withName(statefulSetName).get();
-            assertThat(statefulSet, is(notNullValue()));
-            ResourceRequirements requirements = statefulSet.getSpec().getTemplate().getSpec().getContainers().get(0).getResources();
-            if (resources != null && resources.getRequests() != null) {
-                assertThat(requirements.getRequests(), hasEntry("cpu", resources.getRequests().get("cpu")));
-                assertThat(requirements.getRequests(), hasEntry("memory", resources.getRequests().get("memory")));
-            }
-            if (resources != null && resources.getLimits() != null) {
-                assertThat(requirements.getLimits(), hasEntry("cpu", resources.getLimits().get("cpu")));
-                assertThat(requirements.getLimits(), hasEntry("memory", resources.getLimits().get("memory")));
```

Comment:
I removed it since when I was discussing about the changes with Jakub on the caht, we discussed that the resources play no role with the Mockkube so they were not of much use